src/components/MovieCard/MovieCard.scss

```diff
@@ -1,10 +1,34 @@
 .movie-card {
+  height: 12em;
   width: 20em;
   margin: 2em;
+  overflow: hidden;
+  padding: 0;
   img {
-    width: 100%;
+    object-fit: cover;
+    height: 12em;
+    width: 20em;
+    margin: 0;
+    position: absolute;
   }
   &:hover {
     @include dropShadow;
+    cursor: pointer;
   }
+  &:hover .movie-info-container {
+    height: 4em;
+    bottom: -8em;
+  }
+}
+
+.movie-info-container {
+  background: rgba(255, 255, 255, 0.85);
+  width: 20em;
+  margin: 0;
+  bottom: -12em;
+  position: relative;
+  overflow: hidden;
+  height: 0;
+  transition: .5s ease;
```

Comment:
Great job getting this to work! I think we can fix the few movies that have extra long titles by specifying the top css property instead of bottom for the .movie-info-container. But we can work together on that later. Nice job!